app/components/candidate_interface/other_qualifications_review_component.rb

```diff
@@ -35,7 +35,7 @@ def no_qualification_row
       [{
         key: 'Do you want to add any A levels and other qualifications',
         value: 'No',
-        change_path: candidate_interface_other_qualification_type_path,
+        change_path: candidate_interface_other_qualification_type_path(:change_link),
```

Comment:
@stevehook / @davidgisbey this feels hacky - but it does appear to be working - are there any alternatives to implement this in a cleaner way?

Reply:
I think using the `format` param is bit confusing. How about a simple URL param so you end up with something like `path/to/form?no_other_qualification=true`? Maybe `candidate_interface_other_qualification_type_path(no_other_qualification: true)`?

Reply:
(The type form doesn't need to know anything about change links etc. it just needs to know whether to pre-populate the radio button so the name should reflect that rather than external details)

Reply:
@davidgisbey can you think of a better way to pass this switch from the review page to type form?

Reply:
Query string doesn’t appear to be working (happy for this to be the approach for showing/hiding the last radio option).

Can we use hyphens in the param?

And maybe the param is `?change=true`